src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskCompletionTest.java

```diff
@@ -413,16 +411,6 @@ void given_event_ids_should_evaluate_ReferJudge_dmn(String eventId, List<Map<Str
         MatcherAssert.assertThat(dmnDecisionTableResult.getResultList(), is(expectation));
     }
 
-    @ParameterizedTest(name = "event id: {0}")
-    @MethodSource({"scenarioSettleClaimPaidInFull"})
-    void given_event_ids_should_evaluate_SettleClPaidFull_dmn(String eventId, List<Map<String, String>> expectation) {
-
-        VariableMap inputVariables = new VariableMapImpl();
-        inputVariables.putValue("eventId", eventId);
-        DmnDecisionTableResult dmnDecisionTableResult = evaluateDmnTable(inputVariables);
-        MatcherAssert.assertThat(dmnDecisionTableResult.getResultList(), is(expectation));
-    }
-
```

Comment:
Is the above test not needed. Seems to do with testing of ClaimSettledDivergenceTakeCaseOffline task?

Reply:
Yes, I've removed this independent test and added it to "scenarioProvider()" and also updated it with the new taskType for this PR.

Reply:
Ok, Got you.